afids-auto-train/workflow/Snakefile

```diff
@@ -42,9 +42,37 @@ rule all:
         #    suffix="regqc.html",
         #),
 
+rule skull_strip: 
+    input: 
+    	brainskull = inputs["t1w"].input_path,
+    output: 
+        brain=expand(
+                bids(
+                root=join(config["output_dir"], "skull_striped"),
+                datatype="anat",
+                res="native",
+                suffix="T1w.nii.gz",
+                desc="synthstrip"
+                space=config["space"],
+                **inputs["t1w"].input_wildcards,
+            ),
+        mask=expand(
+                bids(
+                root=join(config["output_dir"], "skull_striped"),
+                datatype="anat",
+                res="native",
+                suffix="brainmask.nii.gz",
```

Comment:
I would change the `desc` to `synthstripbrain` and the suffix to `mask.nii.gz` (in line with https://bids-specification.readthedocs.io/en/stable/05-derivatives/03-imaging.html#masks)